From ece99b6b6df5f05c13719e4873dfac62f6e2a58d Mon Sep 17 00:00:00 2001
From: Julien Olivain <julien.olivain@nxp.com>
Date: Tue, 12 Nov 2019 18:05:52 +0100
Subject: [PATCH 11/37] MLK-22951 mxc: vpu_malone: add missing semicolon

commit bbbbc9c468fdbb9c4d566b9775a98106211b4cff from
https://source.codeaurora.org/external/imx/linux-imx.git

Add a missing semicolon. This missing semicolon is not breaking the
build in defconfig since the next MODULE_DEVICE_TABLE() expand to an
empty string, so the next semicolon is used. When this macro expands
to a non-empty string, the build breaks. This was found by a
"make allmodconfig" build.

Signed-off-by: Julien Olivain <julien.olivain@nxp.com>
Reviewed-by: Shijie Qin <shijie.qin@nxp.com>
Acked-by: Zhou Peng <eagle.zhou@nxp.com>
(cherry picked from commit 0ee54081227ca4eef14ced0a823c40112d2d92e6)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mxc/vpu_malone/vpu_b0.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mxc/vpu_malone/vpu_b0.c b/drivers/mxc/vpu_malone/vpu_b0.c
index f340e6c68e46..e050d126c156 100644
--- a/drivers/mxc/vpu_malone/vpu_b0.c
+++ b/drivers/mxc/vpu_malone/vpu_b0.c
@@ -6679,7 +6679,7 @@ static const struct of_device_id vpu_of_match[] = {
 	{ .compatible = "nxp,imx8qm-b0-vpudec", },
 	{ .compatible = "nxp,imx8qxp-b0-vpudec", },
 	{}
-}
+};
 MODULE_DEVICE_TABLE(of, vpu_of_match);
 
 static struct platform_driver vpu_driver = {
-- 
2.17.1

